icu4c/source/test/intltest/locnmtst.cpp

```diff
@@ -420,6 +421,22 @@ void LocaleDisplayNamesTest::TestRootEtc() {
   delete ldn;
 }
 
+void LocaleDisplayNamesTest::TestNumericRegionID() {
+    UErrorCode err = U_ZERO_ERROR;
+    ULocaleDisplayNames* ldn = uldn_open("es_MX", ULDN_STANDARD_NAMES, &err);
+    UChar displayName[200];
+    int32_t displayNameLength = uldn_regionDisplayName(ldn, "019", displayName, 200, &err);
+    test_assert(U_SUCCESS(err));
+    test_assert_equal(UnicodeString(u"América"), UnicodeString(displayName));
+    uldn_close(ldn);    
+
```

Comment:
OK, then we should reset `err = U_ZERO_ERROR;` here so an error from the "es_MX" test does not affect the "en_AU" test ...

Reply:
I kinda figured one error would just fail out the whole test, but I guess I can do that if you really want it...

Reply:
Well, when resolving errors during CLDR data integration, I kinda hate it when I fix one problem only discover that a new previously-hidden error is now reported...